src/actions/validate.ts

```diff
@@ -16,15 +16,32 @@
 
 import * as core from "@actions/core";
 import { getVersionBumpType } from "../bump";
+import { ConventionalCommitMessage } from "../commit";
 
 import { Configuration } from "../config";
-import { getConfig, isPullRequestEvent, updateSemVerLabel } from "../github";
+import { getConfig, isPullRequestEvent, updateLabels } from "../github";
+import { SemVerType } from "../semver";
 import {
   getMessagesToValidate,
   validateMessages,
   validatePrTitleBump,
 } from "../validate";
 
+/**
+ * Determine labels to add based on the provided conventional commits
+ */
+async function determineLabels(
```

Comment:
Originally I extended this function to also add the `type:` labels to the Pull Request - this was undesired behavior, but decided to keep this abstraction in place.